app/src/configurable_auth.h

```diff
@@ -27,6 +28,10 @@ namespace scitt
       auto identity = JwtAuthnPolicy::authenticate(tx, ctx, error_reason);
       if (!identity)
       {
+        SCITT_INFO(
+          "ClientRequestId={} ErrorMessage={}",
```

Comment:
I think the format should match the log messages emitted for endpoint-originating errors, including the URL etc.

Reply:
add verb and url